endpoints/version_test.go

```diff
@@ -1,52 +1,43 @@
 package endpoints
 
 import (
-	"encoding/json"
 	"io/ioutil"
 	"net/http/httptest"
-	"reflect"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
 )
 
 func TestVersion(t *testing.T) {
-	// Setup:
 	var testCases = []struct {
-		version  string
-		revision string
-		expected string
+		description string
+		version     string
+		revision    string
+		expected    string
 	}{
-		{"", "", `{"version":"not-set","revision":"not-set"}`},
-		{"abc", "def", `{"version":"abc","revision"	:"def"}`},
-		{"1.2.3", "d6cd1e2bd19e03a81132a23b2025920577f84e37", `{"version":"1.2.3","revision":"d6cd1e2bd19e03a81132a23b2025920577f84e37"}`},
+		{
+			description: "Empty",
+			version:     "",
+			revision:    "",
+			expected:    `{"revision":"not-set","version":"not-set"}`,
+		},
+		{
+			description: "Populated",
+			version:     "1.2.3",
+			revision:    "d6cd1e2bd19e03a81132a23b2025920577f84e37",
+			expected:    `{"revision":"d6cd1e2bd19e03a81132a23b2025920577f84e37","version":"1.2.3"}`,
+		},
```

Comment:
Can we include these additional two tests?
```
    {
        description: "Empty",
        version:     "1.2.3",
        revision:    "",
        expected:    `{"revision":"not-set","version":"1.2.3"}`,
    },
    {
    	description: "Populated",
    	version:     "",
    	revision:    "d6cd1e2bd19e03a81132a23b2025920577f84e37",
    	expected:    `{"revision":"d6cd1e2bd19e03a81132a23b2025920577f84e37","version":"not-set"}`,
    },
```

Reply:
Sure.